src/test_snmp.cpp

```diff
@@ -86,6 +88,190 @@ TEST_F(SNMPTest, LatencyCalculations)
   delete tbl;
 }
 
+TEST_F(SNMPTest, LatencyScopeCalculations)
+{
+  cwtest_completely_control_time(true);
+
+  // Create a table.  Use a different OID base because otherwise the indices
+  // clash with later tests.
+  SNMP::EventAccumulatorByScopeTable* tbl = SNMP::EventAccumulatorByScopeTable::create("latency", test_oid);
```

Comment:
You say 'use a different OID base', but then go on to use the same `test_oid` as elsewhere. Am i missing something here?